content/en/news/releases/1.23.x/announcing-1.23.6/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Announcing Istio 1.23.6
+linktitle: 1.23.6
+subtitle: Patch Release
+description: Istio 1.23.6 patch release.
+publishdate: 2025-04-08
+release: 1.23.6
+aliases:
+    - /news/announcing-1.23.6
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.23.5 and Istio 1.23.6
+{{< relnote >}}
+
+## Changes
```

Comment:
TODO: check if i need to include [CVE-2025-30157](https://nvd.nist.gov/vuln/detail/CVE-2025-30157) (CVSS Score 6.5, Medium): Envoy crashes when HTTP ext_proc processes local replies.

Reply:
https://istio.io/latest/news/releases/1.24.x/announcing-1.24.4/#security-updates

Reply:
added a suggestion with it